src/components/Sidenav/SidenavBase.style.ts

```diff
@@ -92,9 +93,10 @@ export const SidebarNavItem = styled.li<ExpandableElementProps>`
   &[data-badge]:after {
     left: ${sizes(10)};
     top: ${sizes(3)};
-    @media screen and (min-width: ${breakpoints.medium}) {
-      transform: translateY(${({ expanded }) => (expanded ? 0 : -8)}px);
-      transition: transform ${transitions.timings.regular} ${transitions.easing};
+
+    ${media.medium} {
+      transform: translateY(${({ expanded }: { expanded?: boolean }) => (expanded ? 0 : -8)}px)
```

Comment:
Types of props are already provided in the component definition `styled.li<ExpandableElementProps>`. We don't need to provide them explicitly here